webapp/package.json

```diff
@@ -46,6 +46,7 @@
     "babel-preset-react": "6.11.1",
     "babel-preset-stage-0": "6.5.0",
     "copy-webpack-plugin": "3.0.1",
+    "cross-env": "^3.0.0",
```

Comment:
Minor, but can you remove the `^`? We prefer not to have our npm packages auto-update since we can never be sure the next versions are stable (we've had issues in the past)